### PR TITLE
Add GET /playlist/:id endpoint

### PR DESCRIPTION
**Pull request by imranolas**
Updated to fetch video items by playlist ID

### CODE REVIEW

File: server/controllers/get-playlist.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+'use strict';
```
Comment (wheresrhys) [inline server/controllers/get-playlist.js]: would you mind renaming this file to playlist to keep the route/controller name parity throughout

**Comment by wheresrhys** [comment]
:+1: aside from the comment above

**Comment by imranolas** [comment]
Thanks

**Pull request merged**